qiskit_addon_cutting/qpd/decompose.py

```diff
@@ -43,6 +42,7 @@ def decompose_qpd_instructions(
         map_ids: Indices to a specific linear mapping to be applied to the decompositions
             in the circuit. If no map IDs are provided, the circuit will be decomposed randomly
             according to the decompositions' joint probability distribution.
+        inplace: TODO
```

Comment:
This needs docstring still

Reply:
Added in 1cce3d845db17d930f3d702e12a4d4691e2e8443.